R/qenv-get_var.R

```diff
@@ -55,3 +55,29 @@ setMethod("[[", signature = c("qenv", "ANY"), function(x, i) {
     class = c("validation", "try-error", "simpleError")
   ))
 }
+
+#' @export
+`$.qenv.error` <- function(x, name) {
```

Comment:
Should we add this method and have it behave just like `[[`?

We need to allow `$` to retrieve existing properties of `qenv.error`, as `conditionMessage(x)` will try to access `x$message`, `x$call` and `x$trace`.

Reply:
Thanks for figuring this out 👍